src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java

```diff
@@ -96,7 +94,7 @@ protected void registerGoals() {
         this.goalSelector.addGoal(9, new GoToWaterGoal(this, 0.8D));
     }
 
-    public static boolean checkGrebeSpawnRules(EntityType<LittleGrebeEntity> animal, LevelAccessor worldIn, MobSpawnType reason, BlockPos pos, RandomSource randomIn) {
+    public static boolean checkGrebeSpawnRules(EntityType<LittleGrebeEntity> animal, LevelAccessor worldIn, MobSpawnType reason, BlockPos pos, Random randomIn) {
```

Comment:
I'm not sure why any changes are being made to this class. It's not relevant to the PR.

Reply:
After looking at the other PR, there is a temptation tag relevant to this class, but you don't make this change here and instead are changing irrelevant code. Probably just some weird mistake with the cherry-picking!

Reply:
Yes that'll be the case, i thought i checked everything so my bad